lib/dynamics_crm/client.rb

```diff
@@ -141,11 +141,10 @@ def rollup(target_entity, query, rollup_type="Related")
         })
     end
 
-    def retrieve_multiple(entity_name, criteria=[], columns=[])
-
+    def retrieve_multiple(entity_name, criteria=[], columns=[], operator=nil)
```

Comment:
This method signature is getting a little complicated. I may want to refactor this to named parameters or an object that can wrap up all the search criteria.